environments/64-test/public.yml

```diff
@@ -0,0 +1,177 @@
+# For a real deployment
+# should be e.g. 'www.commcarehq.org'
```

Comment:
looking at how this is used I don't think it should ever by `www.commcarehq.org`. It mostly gets used to set machine hostnames